package.json

```diff
@@ -73,6 +73,8 @@
     "@kbn/pm": "link:packages/kbn-pm",
     "@kbn/test-subj-selector": "link:packages/kbn-test-subj-selector",
     "@kbn/ui-framework": "link:packages/kbn-ui-framework",
+    "@types/mustache": "^0.8.31",
```

Comment:
Can you please move this to devDependencies so we don't ship in the build?

Reply:
#24165 (sorry about that)

Reply:
Oh no worries, thanks for the quick turn around.